deploy/custom_resources/custom_authorizer/auth_services.py

```diff
@@ -15,6 +22,10 @@ def generate_resource_str(resource, api_resource_name):
         resource_list[5] = '/'.join(api_gateway_arn)
         return ':'.join(resource_list)
 
+    @staticmethod
+    def generate_deny_policy(incoming_resource_str: str):
+        return AuthServices.generate_policy({'sub': None, EMAIL_CLAIM: None}, 'Deny', incoming_resource_str)
```

Comment:
do we need to add USER_ID: None to this dict too? - i.e. `{'sub': None, EMAIL_CLAIM: None, USER_ID: None}`

Otherwise do we run the risk of getting KeyError when updating context as `'user_id': verified_claims[USER_ID_CLAIM],`?

Reply:
I could yes but currently it's working (I tested it) because both `email` and `user_id` env variables are equal to `email`